### DISCUSSION TITLE
Restrict Hubs for certain IP

### DISCUSSION THREAD

**Discussion by bemyevent** [mozilla/hubs#3135] [Hubs Cloud Questions] [answered]
Hello, 

I need to restrict the access of the room of my aws hubs at few range of IP address for a customer . So only the users who have the IP address could access to the room . 

Should I do that when I start to launch the pile in cloudformation  ? 
thx for your reply .

**Comment by robinkwilson** [comment, marked as answer]
Currently this isn't an option in the cloudformation template or on the Hubs side. The way we limit access is through these options: https://hubs.mozilla.com/docs/hubs-cloud-limiting-user-access.html 
OR with custom settings on a custom client: https://hubs.mozilla.com/docs/hubs-cloud-custom-clients.html

Although I haven't done this myself, you may be able to do this after deploying the stack on the AWS side with custom IAM settings for accessing your EC2 instance. With AWS HC, the infrastructure is yours to do what you need. 

Good luck! We'd love to hear what you've found.

**Reply by bemyevent** [reply]
I thought it was in cloudformation just in this section ? 
_**Advanced - Leave these alone unless you know what you're doing!
Inbound Site Access CIDR
IP CDIR which will be granted HTTPS and WebRTC access to your site. Set to 0.0.0.0/0 to allow anyone on the Internet to access your site.**_

**Reply by robinkwilson** [reply]
Ah good! Yes give it a try, I didn't realize that was in the template, my mistake. We don't have docs surrounding those Advanced Settings yet. Please give it a try.
